gen/generate.go

```diff
@@ -83,7 +84,11 @@ func setupGenerateGenesisFiles(genesisData *GenesisData, consensus config.Consen
 		genesisData.FeeSink = defaultSinkAddr
 	}
 	if (genesisData.RewardsPool == basics.Address{}) {
-		genesisData.RewardsPool = defaultPoolAddr
+		if genesisData.DevMode {
+			genesisData.RewardsPool = zeroAddr
+		} else {
+			genesisData.RewardsPool = defaultPoolAddr
+		}
```

Comment:
These shouldn't be needed if the below is done, right?

Reply:
you're probably right, i dont need to set zero addr explicitly here if i set the balance to minbal, will change